_groups/tuhh.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page_group
+title: "Hamburg University of Technology"
+date: 2015-09-01 16:00:00 +0100
+navbar: Groups
+subnavbar: leeds
+logo: uni_leeds.png
```

Comment:
You may want to get rid of Leeds here, too.

Reply:
When you are right, you are right :)